src/camera/Camera.ts

```diff
@@ -637,7 +687,7 @@ abstract class Camera {
     }));
   }
 
-  protected _checkTranslateSupport = () => {
+  private _checkTranslateSupport = () => {
```

Comment:
(protected to private) It would be nice if it wasn't exposed to the outside.

Reply:
As it's `protected` it was open to its child classes(`LinearCamera` and so).
But as they're all gone now, it's bound to the `Camera` only.